src/send-and-sync.md

```diff
@@ -74,7 +74,114 @@ of their pervasive use of raw pointers to manage allocations and complex ownersh
 Similarly, most iterators into these collections are Send and Sync because they
 largely behave like an `&` or `&mut` into the collection.
 
+[`Box`][box-doc] is implemented as it's own special intrinsic type by the
```

Comment:
Could you add an h2 headering here? I think this should be separated.

Reply:
Fixed in e542c32